Makefile

```diff
@@ -86,20 +86,19 @@ clean: clean-e2e-test
 deploy:
 	kubectl apply -k deploy/base
 
-## Build e2e test binary
-.PHONY: build-e2e-test
-build-e2e-test:
-	go test -c ./test/e2e -o _output/e2e.test
-
 ## Runs e2e test
 .PHONY: e2e-test
-e2e-test: build-image build-e2e-test
-	@build/run-e2e-tests.sh
+e2e-test: build-image
```

Comment:
By separate script, we can reuse `setup-ocm.sh` and `setup-import-controller.sh` when we want to test against OCP env.